test/controllers/error.controller.spec.ts

```diff
@@ -0,0 +1,41 @@
+jest.mock("../../src/utils/logger");
+jest.mock("../../src/controllers/landing.controller");
+
+import request from "supertest";
```

Comment:
It is not required but I would have added an implicit import of jest methods, `import { describe, expect, test, beforeEach, jest } from '@jest/globals';`